### ISSUE TITLE
Failure to insert rows to MySQL

### ISSUE THREAD

**Issue by SecurityInsanity** [instructure/canvas-data-loader#14]
`[root@localhost home]# RUST_LOG=info ./target/release/cdl-runner`

```
Error: 
 INFO 2018-11-13T16:42:52Z: cdl_runner: Setting up API Client...
 INFO 2018-11-13T16:42:52Z: cdl_runner: Connecting to RocksDB Store....
 INFO 2018-11-13T16:42:52Z: cdl_runner: Skipping dump: f2861f59-f11c-4c97-8d09-c349e7f4f3f6 
 INFO 2018-11-13T16:42:52Z: cdl_runner: Skipping dump: 259dfc10-69b7-4f9f-bd89-013082e6c1ef due to only final selected
 INFO 2018-11-13T16:42:52Z: cdl_runner: Skipping dump: a8d4271f-2013-4ff0-8813-b8aa79e85e0b due to only final selected
 INFO 2018-11-13T16:42:53Z: cdl_runner: Connecting to the DB
ERROR 2018-11-13T16:50:35Z: cdl_runner::db_client: insert error
ERROR 2018-11-13T16:50:35Z: cdl_runner::db_client: MySqlError { ERROR 1264 (22003): Out of range value for column 'position' at row 1 }
ERROR 2018-11-13T16:50:35Z: cdl_runner::importer: process -> for line in finalized_string -> !is_volatile -> ins_res -> is_err
ERROR 2018-11-13T16:50:35Z: cdl_runner::importer: Error(MysqlErr, State { next_error: None, backtrace: None })
 INFO 2018-11-13T16:50:35Z: cdl_runner: Done!
```
My question is: Is there a fix for this?

_Originally posted by @latifatbraimah in https://github.com/instructure/canvas-data-loader/issues/6#issuecomment-438354072_

**Comment by SecurityInsanity** [comment]
Sorry for directing you back to that issue. Seems there was a mix up in filing a github issue. We'll talk about it here @latifatbraimah . The canvas-data-loader itself does not pad any values. Are these values present in the source dumps? (e.g. if you download them manually from the UI do you find these padded values).

**Comment by latifatbraimah** [comment]
No. When I do a manual download and unpack, upload is pretty straightforward. But it's also very tedious especially with 80+ relations. This is why I'm using canvas-data-loader

**Comment by SecurityInsanity** [comment]
How are you uploading? Again we don't do any padding on our end, so I feel like this is coming from the source data. Can you look through the source data for these 5's? A simple grep should be all that's needed.

**Comment by latifatbraimah** [comment]
I use `LOAD DATA LOCAL INFILE` and No. Just checked my downloads, no padding with 5s

**Comment by SecurityInsanity** [comment]
Hmm canvas data loader should pretty much be doing the same thing. Can you let me know what school your with so I can check your data? Also what version of MySQL are you using? 

If you don't feel comfortable giving out school info on this public forum you can always file a support case by emailing canvasdatahelp@instructure.com ,and we can work through it there as well.
